aten/src/ATen/SparseCsrTensorImpl.h

```diff
@@ -57,6 +57,22 @@ struct TORCH_API SparseCsrTensorImpl : public TensorImpl {
     return col_indices_.size(-1);
   }
 
+  int64_t batch_dim() const {
```

Comment:
nit: this could be marked with `inline ... const noexcept` for a strong inline hint.

Reply:
applied